core/scrollbar.js

```diff
@@ -1079,26 +773,26 @@ Blockly.Scrollbar.prototype.onMouseUpHandle_ = function() {
  * wrap up loose ends associated with the scrollbar.
  * @private
  */
-Blockly.Scrollbar.prototype.cleanUp_ = function() {
+Scrollbar.prototype.cleanUp_ = function() {
   Blockly.hideChaff(true);
```

Comment:
Can you confirm that Blockly.hideChaff is correct?  It doesn't appear to be imported, but I'm not confident in my reading of this.

Reply:
This file should have `require('Blockly');` but it causes a circular dependency.
The require is currently commented out at the top of the file (with a reference to the issue number).

I have tested by running the playground, setting a breakpoint here, and calling `Blockly.getMainWorkspace().dispose()` and see that this line runs fine (and calls the correct method.